lib/commands/web.js

```diff
@@ -215,7 +215,7 @@ extensions.translateWebCoords = async function translateWebCoords (coords) {
   try {
     this.setImplicitWait(0);
     webview = await retryInterval(5, 100, async () => {
-      return await this.findNativeElementOrElements('-ios predicate string', `type = 'XCUIElementTypeWebView' AND visible = 1`, false);
+      return await this.findNativeElementOrElements('class-name', 'XCUIElementTypeWebView', false);
```

Comment:
This should be `class name` (https://github.com/appium/appium-xcuitest-driver/blob/master/lib/driver.js#L133)

Reply:
Not sure how I missed that. Thanks.